src/init/defineData/deriveVariables.js

```diff
@@ -0,0 +1,18 @@
+export default function deriveVariables() {
+    this.data.raw.forEach(d => {
+        d.brushed = false;
+        if (d[this.config.unit_col])
+            d.measure_unit = `${d[this.config.measure_col]} (${d[this.config.unit_col]})`;
+        else d.measure_unit = d[this.config.measure_col];
+
+        const lo =
+                d[this.config.lln_col] !== undefined
+                    ? +d[this.config.value_col] < +d[this.config.lln_col]
+                    : false,
```

Comment:
We should consider imputing  upper and lower limits when none are given in the data. Would be nice to provide a control that lets users define outliers using either Standard deviations or Quantiles. We can handle this in a separate issue.